config/cobbler/distro_signatures.json

```diff
@@ -1035,7 +1035,7 @@
     "kernel_file":"linux[64.gz]?",
     "initrd_file":"initrd[64]?",
     "isolinux_ok":false,
-    "default_kickstart":"/var/lib/cobbler/kickstarts/sample_autoyast.xml",
+    "default_autoinstall":"sample_autoyast.xml",
```

Comment:
Looks sane, but as mentioned above: There are a lot of these and they should be fixed all and in a separate commit.